ui/admin/app/components/form/credential-store/static/index.hbs

```diff
@@ -68,28 +68,22 @@
               @layout='wide'
             />
           </form.radioGroup>
-        {{else}}
-          <form.radioGroup
-            @name='types'
-            @selectedValue={{@model.type}}
-            as |radioGroup|
-          >
-            <radioGroup.card
-              @label={{t
-                (concat 'resources.credential-store.types.' @model.type)
-              }}
-              @helperText={{t
-                (concat 'resources.credential-store.help.' @model.type)
-              }}
-              @value={{@model.type}}
-              @layout='wide'
-              readonly={{true}}
-              @disabled={{true}}
-            />
-          </form.radioGroup>
-        {{/if}}
-      </:body>
-    </form.fieldset>
+        </:body>
+      </form.fieldset>
+    {{else}}
+      <InfoField
+        @value={{t (concat 'resources.credential-store.types.' @model.type)}}
+        disabled={{form.disabled}}
```

Comment:
Should InfoField be disabled by default? Even when editing, we do not allow resources to change their type.